es6/experiment.js

```diff
@@ -100,7 +100,20 @@ class Experiment {
   }
 
   assign(params, args) {
-    throw "IMPLEMENT THIS";
+    throw "IMPLEMENT assign";
+  }
+
+  experimentParameters() {
+    throw "IMPLEMENT experimentParameters";
+  }
+
+  shouldFetchExperimentParameters(name) {
+    let experimentalParams = this.experimentParameters();
+    if (typeof experimentalParams == "boolean") {
```

Comment:
I wanted to leave open the possibility of allowing users to be able to just always set this to be true in case the range of experimental parameters can't be easily defined, though the more I think about this the more it appears to be a bad practice that should not be encouraged. 

@TimBeyer do you have any thoughts on this?